docs/src/constructors.md

```diff
@@ -260,9 +210,6 @@ DisplayAs.Text(ans) # hide
 ## Fitting generalized linear mixed models
 
 To create a GLMM representation
-```@docs
-GeneralizedLinearMixedModel
-```
 the distribution family for the response, and possibly the link function, must be specified.
```

Comment:
I can't make a suggestion on a deleted line, but can you remove the linebreak and add a comma?

Reply:
Incorporated in a separate commit.